dolosse/generic_json_consumer.py

```diff
@@ -0,0 +1,142 @@
+"""Reads json data from a kafka topic and graphs it"""
+import json
+import time
+import sys
+import curses
+from matplotlib import pyplot
+import kafka
```

Comment:
Imports should be grouped according to 
- standard imports
- third party imports
- project imports

One blank line between imports, alphabetical within their groups. Favor `from X import` instead of importing the full library. 

```python
from json import decode
from sys import argv
from time import 

from curses import
from kafka import
from matplotlib import pyplot
```

This will help ensure that you don't include more than you intended, and that you don't override standard library names by accident

Reply:
Fixed.